### PR TITLE
Parse alternative phrasing blocks

### PR DESCRIPTION
**Pull request by mruwnik**
https://github.com/StampyAI/GDocsRelatedThings/issues/29

### CODE REVIEW

File: parser/constants.js [hunk 1]
```diff
@@ -19,4 +19,5 @@ export const codaColumnIDs = {
   suggestionSize: "c-6DnuBdIZ02",
   UIID: "c-J0hTr2p6-T",
   status: "c-Gr2GDh30nR",
+  alternativePhrasings: "c-bLWz7lnXHG",
```
Comment (Aprillion) [inline parser/constants.js:22]: you know you can make a PR with 2 commits without merge conflicts between your own changes on the same line, right?

File: parser/parser.js [hunk 2]
```diff
@@ -8,6 +8,72 @@ const extractBlockText = (block) =>
     .map((text) => text.trim())
     .join("");
 
+const extractFootnotes = (documentContext, doc) =>
+  Object.keys(documentContext.footnotes)
+    .map((fnID) => {
+      return (
```
Comment (Aprillion) [inline parser/parser.js:14]: prettier didn't want to convert this to implicit return?

File: parser/__tests__/parser.test.js [hunk 3]
```diff
@@ -424,6 +424,58 @@ describe("parseDoc", () => {
...
+    expect(result.md).toEqual("This is some text\n\n");
+    expect(result.alternativePhrasings).toEqual([
+      "some other way of saying things",
+      "bla bla bla",
+    ]);
+    expect(result.suggestionCount).toEqual(0);
+    expect(result.suggestionSize).toEqual(0);
+  });
+
+  it("handles alternative phrasings even when after related section", async () => {
```
Comment (Aprillion) [inline parser/__tests__/parser.test.js:451]: 👍

**Review by Aprillion — APPROVED** [review]

File: parser/constants.js (hunk 1 above)
Comment (mruwnik) [reply parser/constants.js:22]: yup. But I prefer to have separate ones when there's lot's of changes

**Pull request merged**